# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/slrn/slrn-0.9.8.0-post.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- slrn-0.9.8.0/src/post.c	2003-07-31 19:21:18.000000000 +0200
+++ slrn/src/post.c	2003-09-04 23:56:33.000000000 +0200
@@ -1233,6 +1234,9 @@
 
    va_start(ap, fmt);
    Slrn_Post_Obj->po_vprintf(fmt, ap);
+   va_end(ap);
+
+   va_start(ap, fmt);
    vfprintf(fcc_fp, fmt, ap);
    va_end(ap);
 }
